src/main/java/io/cdap/plugin/servicenow/source/ServiceNowRecordReader.java

```diff
@@ -159,7 +162,7 @@ private void fetchSchema(ServiceNowTableAPIClientImpl restApi) {
     schema = Schema.recordOf(tableName, schemaFields);
   }
 
-  private Object convertToValue(String fieldName, Schema fieldSchema, Map<String, Object> record) {
+  public Object convertToValue(String fieldName, Schema fieldSchema, Map<String, Object> record) {
```

Comment:
use protected instead of public, if you are using it only in child class.

Reply:
Changed access modifier from `public` to `protected` 7661e08